src/Management/src/EndpointBase/Metrics/IMetricsObserverOptions.cs

```diff
@@ -19,6 +19,15 @@ public interface IMetricsObserverOptions
     /// </summary>
     string EgressIgnorePattern { get; }
 
+    /// <summary>
+    /// Gets Allow list of metrics that should be captured
+    /// </summary>
+    /// <remarks>
+    ///     Currently only applies to System.Runtime metrics captured by <see cref="EventCounterListener"/><para />
+    ///     See this list for values to choose from: <see href="https://docs.microsoft.com/dotnet/core/diagnostics/available-counters#systemruntime-counters" />
+    /// </remarks>
```

Comment:
I suspect this is going to fail in the main branch because of [SA1629](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1629.md).

Reply:
For summary, remarks or both? Want to add it as a suggestion?

Reply:
From the link I posted:
> A violation of this rule occurs when part of the documentation does not end with a period. 

> How to fix violations
> To fix a violation of this rule, ensure that all sections of the documentation end with a period.

Reply:
I will fix this in main branch.